src/harness/runner.ts

```diff
@@ -132,8 +133,8 @@ function handleTestConfig() {
             return true;
         }
 
-        if (testConfig.test && testConfig.test.length > 0) {
-            for (const option of testConfig.test) {
+        if (testConfig.runners && testConfig.runners.length > 0) {

```

Comment:
should probably use `runners||test` so that `jake runtests t=rwc` still works.

Reply:
Done; now the CI server won't break :P